pladder/log.py

```diff
@@ -140,6 +153,42 @@ def add_line(self, timestamp, channel, nick, text):
             c.execute("INSERT INTO lines (timestamp, channel, nick, text) VALUES (?, ?, ?, ?);",
                       (timestamp, channel, nick, text))
 
+    def mimic(self, channel, nick):
+        stats = self._generate_mimic_stats(channel, nick)
+        return self._generate_mimic_line(stats)
+
+    def _generate_mimic_stats(self, channel, nick):
```

Comment:
It will be interesting to see how the performance of this is... :-) Let's find out!

Reply:
I was thinking of caching the statistics somehow, but wanted to try it out first :)